geopandas/tests/test_plotting.py

```diff
@@ -516,7 +516,7 @@ def test_style_kwargs_linestyle(self):
             self.df.plot(linestyle=ls, linewidth=1),
             self.df.plot(column="values", linestyle=ls, linewidth=1),
         ]:
-            np.testing.assert_array_equal(exp_ls, ax.collections[0].get_linestyle())
+            exp_ls == ax.collections[0].get_linestyle()
```

Comment:
```suggestion
            assert exp_ls == ax.collections[0].get_linestyle()
```
Without `assert` this doesn't raise, does it?

Reply:
Yes, of course ;)